corpus_studies/solidity_state_study/slither_plugins/README

```diff
@@ -0,0 +1,2 @@
+To use these plugins with slither, first follow the instructions at https://github.com/crytic/slither/wiki/Developer-installation to set up the slither development environment. 
```

Comment:
__pycache__ files appear to be derived binaries; remove them from the PR. You can add them to .gitignore.